pandas/core/arrays/base.py

```diff
@@ -680,7 +682,12 @@ def argmax(self, skipna: bool = True) -> int:
             raise NotImplementedError
         return nargminmax(self, "argmax")
 
-    def fillna(self, value=None, method=None, limit=None):
+    def fillna(
+        self,
+        value: Any | ArrayLike | None = None,
+        method: Literal["backfill", "bfill", "ffill", "pad"] | None = None,
```

Comment:
are these consistent with the Series signatures?

Reply:
will update that.